packages/vaadin-grid-pro/src/vaadin-grid-pro-inline-editing-mixin.d.ts

```diff
@@ -1,11 +1,9 @@
-import { GridItem } from '@vaadin/vaadin-grid';
-
-declare function InlineEditingMixin<T extends new (...args: any[]) => {}>(base: T): T & InlineEditingMixinConstructor;
-interface InlineEditingMixinConstructor {
-  new (...args: any[]): InlineEditingMixin;
+declare function InlineEditingMixin<TItem, T extends new (...args: any[]) => {}>(base: T): T & InlineEditingMixinConstructor<TItem>;
```

Comment:
nit: let's add a newline here and consider formatting the code as well:

```suggestion
declare function InlineEditingMixin<TItem, T extends new (...args: any[]) => {}>(
  base: T
): T & InlineEditingMixinConstructor<TItem>;

```